pkg/controller/provider/web/base/client.go

```diff
@@ -44,6 +42,8 @@ type Client struct {
 	Host string
 	// Parameters
 	Params Params
+	// http Transport.
+	transport http.RoundTripper
```

Comment:
Shouldn't it be `Transport` with an uppercase `T`?

Reply:
The uppercase `Transport` would result in the filed being _exported_ (public outside the package).  The purpose of the field is to cache the transport internally so that it is only created once not so that it can be supplied on construction.  However, if you think there is a valid use case for the _Transport_ to be supplied, I don't see any potential danger in supporting it.